data-syncs/building-data-syncs/README.md

```diff
@@ -1,161 +1,138 @@
-# Build data syncs
+# Build data syncs in Cinchy
 
-## Guide
+Data syncs in Cinchy synchronizes information between different systems. This guide walks you through the process of setting up your data syncs.
 
-You can use this checklist in conjunction with the documentation below and elsewhere in this space to configure your data syncs in Cinchy.
+## Basic workflow
 
-- [ ] [Install any applicable components](../installation-and-maintenance/)
-- [ ] Determine your [Data Sync Type](types-of-data-syncs.md) and [Design Pattern](common-design-patterns.md)
-- [ ] Define and Configure your [Data Sync Source](../supported-data-sync-sources/)
-- [ ] Define and Configure your [Data Sync Destination](../supported-data-sync-destinations/)
-- [ ] Define and Configure your [Sync Actions](sync-actions.md)
-- [ ] Define and Configure your [Event Stream source](../supported-real-time-sync-stream-sources/) (if running a real-time sync)
+The basic workflow of a data sync in Cinchy is:
 
-## Prerequisites
+1. [Install Necessary Components](link-to-installation-doc).
+1. [Choose Data Sync Type and Design Pattern](link-to-sync-type-pattern).
+1. [Define and Configure Data Sync Source](link-to-configure-source).
+1. [Set Up Data Sync Destination](link-to-setup-destination).
+1. [Specify Sync Actions](link-to-sync-actions).
+1. [Configure Event Stream Source (For Real-time Syncs)](link-to-event-stream).
 
-Before you start, make sure you've installed all the necessary components that you need for your data sync.
+### Overview
 
-## Overview
+You have two ways to set up a data sync in Cinchy:
 
-When you've decided on the type of data sync you want to use, you're ready to configure it.
+1. Use the Connections UI to input and save configuration details. The data will be stored as an XML file in the **Data Sync Configurations** table.
+2. Directly upload an XML config into the **Data Sync Configurations** table.
 
-You have two options when you create a data sync in Cinchy.
+For real-time syncs, you'll also need to set up a lister configuration in the **Listener Configuration** table.
```

Comment:
For real-time syncs, you'll also need to set up a lister configuration in the **Listener Configuration** table, _if you're using the UI_.

Reply:
For real-time syncs using Connections UI, you'll also need to set up a lister configuration in the **Listener Configuration** table.

Reply:
Also missed the misuse of **Listener Conifguration** when it's **Listener Config**

Reply:
Not sure how I missed this... in 5.7 if you use the UI, you don't need to touch the listener config table at all. @balmasi not sure if you want to mention that users _can_ change data in the listener config table too, if they want.

Reply:
I'd love to remove this and have it under an advanced settings.